File: summer/WebContent/js/system/windows/input/SynchronizedInputStates.js
/**
 * SynchronizedInputStates
 */
define(["dojo/_base/declare"], function(declare){

    var SynchronizedInputStates=declare("SynchronizedInputStates", null,{ 
    });	
    
	  
    SynchronizedInputStates.NoOpportunity   = 1; 
    SynchronizedInputStates.HadOpportunity  = 2;
    SynchronizedInputStates.Handled         = 3; 
    SynchronizedInputStates.Discarded       = 4; 
    
    return SynchronizedInputStates;
}); 

File: org.summer.view.widget/src/org/summer/view/widget/internal/SynchronizedInputStates.java
package org.summer.view.widget.internal;
/*internal*/ public enum SynchronizedInputStates
{ 
    NoOpportunity  ,//= 0x01, 
    HadOpportunity ,//= 0x02,
    Handled        ,//= 0x04, 
    Discarded      ,//= 0x08
};